packages/docs/pages/_meta.json

```diff
@@ -1,15 +1,8 @@
 {
   "index": "Namada",
   "introduction": "Introduction",
-  "ledger": "Full nodes",
-  "validators": "Validators",
-  "delegators": "Delegators",
-  "users": "(Power) Users",
-  "public-goods-stewards": "Public goods stewards",
-  "proof-of-stake": "Cubic Proof-of-Stake",
-  "governance": "Governance",
-  "wallet": "Wallet guide",
-  "experimental": "Experimenting with Namada",
-  "troubleshooting": "Troubleshooting",
-  "faq": "FAQ"
+  "users": "User Guide",
+  "operators": "Operator Guide",
+  "integrating-with-namada" : "Integration Guide",
+  "testnets": "Testnets"
```

Comment:
I think we should call it `Networks` and then have sub-categories for `Mainnets` and `Testnets`.

Our chain-id should be prefixed with `main` and `test`

Reply:
Oo I like this suggestion. I'll add it